src/frontend/scss/_main.scss

```diff
@@ -20,116 +20,21 @@
-@import './objects/buttons';
-@import './objects/characteristics';
-@import './objects/react-autosuggest';
-@import './objects/search-filter-value';
-@import './objects/selector';
-@import './objects/form';
-@import '../js/components/AddressesManagement/styles';
-@import '../js/components/CourseProductCertificateItem/styles';
-@import '../js/components/CourseProductCourseRuns/styles';
-@import '../js/components/CourseProductItem/styles';
-@import '../js/components/CourseRunEnrollment/styles';
-@import '../js/components/LtiConsumer/styles';
-@import '../js/components/Modal/styles';
-@import '../js/components/PaymentButton/styles';
-@import '../js/components/RegisteredAddress/styles';
-@import '../js/components/RegisteredCreditCard/styles';
-@import '../js/components/SaleTunnel/styles';
-@import '../js/components/SaleTunnelStepPayment/styles';
-@import '../js/components/SaleTunnelStepResume/styles';
-@import '../js/components/SaleTunnelStepValidation/styles';
-@import '../js/components/SearchFilterGroup/styles';
-@import '../js/components/SearchFilterGroupModal/styles';
-@import '../js/components/SearchFiltersPane/styles';
-@import '../js/components/SearchInput/styles';
-@import '../js/components/Spinner/styles';
-@import '../js/components/StepBreadcrumb/styles';
-@import '../js/components/UserLogin/styles';
-@import '../js/components/UserMenu/styles';
-@import './components/templates/courses/plugins/category_plugin';
-@import './components/templates/courses/plugins/licence_plugin';
-@import './components/templates/richie/section/section';
-@import './components/templates/richie/large_banner/large_banner';
-@import './components/templates/richie/nesteditem/nesteditem';
-@import './components/templates/richie/glimpse/glimpse';
+@import './objects/all';
+
+@import '../js/components/all';
 
-// Page component styles
-@import './components/header';
-@import './components/subheader';
-@import './components/footer';
-@import './components/styleguide';
-@import './components/error';
-@import './components/content';
-@import './components/templates/search/search';
-@import './components/templates/richie/single-column';
-@import './components/templates/richie/multiple-columns';
-@import './components/templates/courses/cms/homepage';
-@import './components/templates/courses/cms/organization_list';
-@import './components/templates/courses/cms/organization_detail';
-@import './components/templates/courses/cms/category_list';
-@import './components/templates/courses/cms/category_detail';
-@import './components/templates/courses/cms/course_detail';
-@import './components/templates/courses/cms/person_list';
-@import './components/templates/courses/cms/person_detail';
-@import './components/templates/courses/cms/blogpost_list';
-@import './components/templates/courses/cms/blogpost_detail';
-@import './components/templates/courses/cms/program_list';
-@import './components/templates/courses/cms/program_detail';
+@import './components/all';
```

Comment:
Not a fan to have this file at the root of js/components/ folder.

Furthermore, within scss/ folder, we have a components/ directory so, from my point of view, it seems logic that those styles are imported through components/_all.scss.

What do you think? Do you see drawbacks ?

Reply:
Ok. Done.